app/views/spree/shared/_taxonomies.html.erb

```diff
@@ -1,16 +1,43 @@
-<% max_level = Spree::Config[:max_level_in_taxons_menu] || 1 %>
+<%# max_level = Spree::Config[:max_level_in_taxons_menu] || 1 %>
 
-<nav id="taxonomies" class="sidebar-item" data-hook aria-label="taxonomies">
-  <% @taxonomies.each do |taxonomy| %>
-    <% cache [I18n.locale, taxonomy, max_level, @taxon] do %>
-      <% if taxonomy.root.children.any? %>
+<!-- <nav id="taxonomies" class="sidebar-item" data-hook aria-label="taxonomies">
```

Comment:
id and data hook is missing in below code. please justify why its missing?

id="taxonomies" , data-hook aria-label="taxonomies"